### DISCUSSION TITLE
How i get the ID Group

### DISCUSSION THREAD

**Discussion by RedeAlphaFitness** [devlikeapro/waha#712] [Q&A]
I'm trying to get the group ID to add members to a group X, but I can't retrieve group IDs, and only the group creation feature works. I don't know if I need to pay for Waha Plus or if the service is unavailable.

**Comment by devlikepro** [comment]
Hi!
1. What does `GET /api/{session}/groups` return? https://waha.devlike.pro/docs/how-to/groups/ The group is one with format like `123123@g.us`
2. Do you use latest WAHA version?
3. What engine do you use? It's WEBJS by default

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by RedeAlphaFitness** [reply]
I'm trying directly through Swagger because when I was attempting from my machine, the results weren't good.

**Reply by RedeAlphaFitness** [reply]
My server is up on Railway, I just can't get group IDs to automatically add participants to a specific group.

**Comment by devlikepro** [comment]
>  I can't retrieve group IDs

Is there any errors in docker logs may be or response?

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by RedeAlphaFitness** [reply]
The response I receive is []
